server/src/cg/group4/server/database/query/GetGroup.java

```diff
@@ -22,28 +22,30 @@
      *
      * @param groupId The given group id.
      */
-    public GetGroup(String groupId) {
+    public GetGroup(final String groupId) {
         cGroupId = groupId;
     }
 
     @Override
-    public Group query(DatabaseConnection databaseConnection) throws SQLException {
+    public Group query(final Connection databaseConnection) throws SQLException {
+        String preparedStatement = "SELECT * FROM 'Group' INNER JOIN User ON 'Group'.OwnerId = User.Id "
```

Comment:
Queries should in general be looked on once again, now that we have Remote and Local storage. We'll need to discuss what we are going to store where. Not for this PR.